.gitignore

```diff
@@ -21,3 +21,4 @@ job_data_example/config/
 cliLogs.txt
 controllerLogs.txt
 executorLogs.txt
+testLogs.txt
```

Comment:
for a better convention, logs file should be in `.log` file type such as: 

- controller.log
- executor.log

and also you should put it inside some folder, for example, the logs folder will be in: `/var/log/octavius/controller.log`

this would make our logging following standard UNIX scheme. Can you help refactor this?

Reply:
or we can create new folder in `~/.octavius/log/`

Reply:
Sure. Now all the logs are written in ~/.octavius/log/